packages/manager/apps/cloud/client/app/app.js

```diff
@@ -261,4 +264,16 @@ angular.module('managerApp', [
   .config(/* @ngInject */ (CucConfigProvider, coreConfigProvider) => {
     CucConfigProvider.setRegion(coreConfigProvider.getRegion());
   })
+  .run(/* @ngInject */ ($state) => {
+    $state.defaultErrorHandler((error) => {
+      if (error.type === RejectType.ERROR) {
+        $state.go('telecom.error', {
```

Comment:
Wrong state name I guess because we are in Cloud application here.

Reply:
Copy/Paste is wrong 😄